### PR TITLE
Consolidate resource files per project

### PR DESCRIPTION
**Pull request by paulomorgado**
Some projects have more than one resource (.resx and .(language).xlf) file per project.

This PR consolidates those multiple files into one.

### CODE REVIEW

File: src/Microsoft.CodeQuality.Analyzers/Core/MicrosoftCodeQualityAnalyzersResources.resx [hunk 1]
```diff
@@ -126,4 +126,1221 @@
   <data name="DoNotUseCountWhenAnyCanBeUsedTitle" xml:space="preserve">
     <value>Do not use Count() when Any() can be used</value>
   </data>
+  <data name="AvoidAsyncVoidTitle" xml:space="preserve">
```
Comment (mavasani) [inline src/Microsoft.CodeQuality.Analyzers/Core/MicrosoftCodeQualityAnalyzersResources.resx]: Probably not related to this PR, but all the async related rule prototypes/stubs have been removed from this repo and so the corresponding resource strings are redundant. We should eventually delete them.

File: src/Microsoft.CodeQuality.Analyzers/Core/xlf/MicrosoftCodeQualityAnalyzersResources.cs.xlf [hunk 2]
```diff
@@ -17,6 +17,2031 @@
         <target state="new">Do not use Count() when Any() can be used</target>
         <note />
       </trans-unit>
+      <trans-unit id="AppendConfigureAwaitTrue">
+        <source>Append .ConfigureAwait(true)</source>
+        <target state="translated">Připojit .ConfigureAwait(true)</target>
```
Comment (mavasani) [inline src/Microsoft.CodeQuality.Analyzers/Core/xlf/MicrosoftCodeQualityAnalyzersResources.cs.xlf]: Thank you so much for retaining the translations!!

**Comment by dotpaul** [comment]
> I didn't see some of the security-related resources under Microsoft.NetCore.Analyzers touched, but we can save that work for another PR (and maybe make me or @LLLXXXCCC do it).

Or...I just missed them the first time around.  Glad they're consolidated.  🙂

**Comment by Adawntoremember** [comment]
src/Microsoft.CodeQuality.Analyzers/Core/ApiDesignGuidelines/CollectionPropertiesShouldBeReadOnly.cs

**Pull request merged**